camelAssemblies/src/main/resources/sourceConfigOverrides/jira_add_comment_sink/v3_21_0/jira_add_comment_sink.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This (well, these) are the source overrides.  They include the `repoList` used.  This same list is used for all the JIRA-related assemblies.